website/content/docs/concepts/aliases.mdx

```diff
@@ -0,0 +1,264 @@
+- [Create a new target and alias at the same time](/boundary/docs/concepts/aliases#create-a-new-target-and-alias)
+- [Associate an existing alias with a target](/boundary/docs/concepts/aliases#associate-an-existing-alias-with-a-target)
+
+### Create a new alias for an existing target
+
+You can create a new alias and associate it with an existing target at the same time.
+Complete the following steps to create a new alias and associate it with a target:
+
+<Tabs>
+
+<Tab heading="CLI">
+
+1. Log in to Boundary.
+1. Use the following command to create an alias:
+
+   ```shell-session
+   $ boundary aliases create target \
+       -description 'This is an example alias for target tcp_1234567890' \
+       -destination-id tcp_1234567890
+       -name Example Boundary alias \
+       -scope-id global \
+       -value example.alias.boundary \
+       -authorize-session-host-id hst_1234567890 \
+   ```
+
+   You can use any of the following [attributes](/boundary/docs/concepts/domain-model/aliases) when you create an alias:
+
+   - `-description=<string>` - Specifies the optional description you want to use for identification purposes.
+   - `-destination-id=<string>` - Specifies the ID of the target that the alias references.
+   - `-name=<string>` - Specifies the optional name you want to use to describe the alias for identification purposes.
+   - `-scope-id=<string>` - Scope in which to create the alias. The default is `global`.
+   You can also specify the scope using the BOUNDARY_SCOPE_ID environment variable.
+   At this time, aliases are only supported for the global scope.
+   - `-value=<string>` - Specifies the string that you want to use as the alias to represent the target.
+   The alias `value` can be a hostname or a DNS-like string.
+   - `-authorize-session-host-id=<string>` - Optionally indicates the host ID to use when you use the alias to authorize a session.
+
+</Tab>
+
+<Tab heading="UI">
+
+1. Log in to Boundary.
+1. Select **Aliases** in the navigation pane.
+1. Click **New Alias**.
+1. Complete the following fields:
+   - **Name**: (Optional) Enter an optional name for the alias to use for identification purposes.
+   - **Description**: (Optional) Enter an optional description for the alias to use for identification purposes.
+   - **Type**: Select **Target**.
+   At this time, targets are the only Boundary resources that supports aliasing.
+   - **Alias Value**: Enter the string that you want to use as the alias to represent the target.
+   An alias's value can be a hostname or a DNS-like string.
+   - **Target ID**: (Optional) Specify any targets you want to associate with the alias.
+   - **Host ID**: (Optional) Enter an optional host ID, if you would like to specify that the alias always uses the same host when you use it to connect to a target.
+1. Click **Save**.
+
+</Tab>
+
+</Tabs>
+
+### Create a new target and alias
```

Comment:
I found this header a little confusing, just suggesting different verbage

```suggestion
### Create an alias during target creation
```

Reply:
I like this much better. Thanks!